ledger-core/v2/virtual/testutils/slotmachine/slotmachine_wrapper.go

```diff
@@ -0,0 +1,135 @@
+// Copyright 2020 Insolar Network Ltd.
+// All rights reserved.
+// This material is licensed under the Insolar License version 1.0,
+// available at https://github.com/insolar/assured-ledger/blob/master/LICENSE.md.
+
+package slotmachine
+
+import (
+	"context"
+	"testing"
+	"time"
+
+	"github.com/gojuno/minimock/v3"
+
+	"github.com/insolar/assured-ledger/ledger-core/v2/conveyor/smachine"
+	"github.com/insolar/assured-ledger/ledger-core/v2/instrumentation/inslogger"
+	messageSenderAdapter "github.com/insolar/assured-ledger/ledger-core/v2/network/messagesender/adapter"
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner"
+	"github.com/insolar/assured-ledger/ledger-core/v2/runner/machine"
+	testUtilsCommon "github.com/insolar/assured-ledger/ledger-core/v2/testutils"
+	"github.com/insolar/assured-ledger/ledger-core/v2/vanilla/synckit"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/integration/convlog"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/testutils"
+	"github.com/insolar/assured-ledger/ledger-core/v2/virtual/testutils/messagesender"
+)
+
+type ControlledSlotMachine struct {
```

Comment:
Please add a watchdog, which is activated by default and calls global.Fatal when time is out